libs/ledger-live-common/src/families/stellar/js-createTransaction.ts

```diff
@@ -16,6 +16,10 @@ const createTransaction = (): Transaction => ({
   memoValue: null,
   memoType: null,
   useAllAmount: false,
+  operationType: "payment",
```

Comment:
We have an option in the transaction call mode that does the same thing as operationType I guess, but payment is called send, it's something that would be nice to refacto but not mandatory

Reply:
Just to clarify, should we rename "payment" to "send" for `operationType`?